packages/@o3r/core/schematics/library/rules/rules.ng.ts

```diff
@@ -40,7 +43,13 @@ export function ngGenerateModule(options: NgGenerateModuleSchema & { targetPath:
     return chain([
       mergeWith(templateNg, MergeStrategy.Overwrite),
       updatePackageDependenciesFactory(options.targetPath, otterVersion!, o3rCorePackageJson, options),
-      updateNgPackagrFactory(options.targetPath)
+      updateNgPackagrFactory(options.targetPath),
+      (t) => {
+        const packageJson = t.readJson(path.posix.join(options.targetPath, 'package.json')) as PackageJson;
```

Comment:
`path.posix.join(options.targetPath, 'package.json')` could set in `const`

Reply:
https://github.com/AmadeusITGroup/otter/pull/905